docs/src/userguide/parallelism.rst

```diff
@@ -12,15 +12,26 @@ Using Parallelism
     ../two-syntax-variants-used
 
 Cython supports native parallelism through the :py:mod:`cython.parallel`
-module. To use this kind of parallelism, the :term:`GIL<Global Interpreter Lock or GIL>` must be released
-(see :ref:`Releasing the GIL <nogil>`).
-It currently supports OpenMP, but later on more backends might be supported.
+module.  It currently supports OpenMP, but later on more backends might be supported.
 
 .. NOTE:: Functionality in this module may only be used from the main thread
           or parallel regions due to OpenMP restrictions.
 
 
-.. function:: prange([start,] stop[, step][, nogil=False][, use_threads_if=CONDITION][, schedule=None[, chunksize=None]][, num_threads=None])
+Historically, this kind of parallelism could only be used with the
+:term:`GIL<Global Interpreter Lock or GIL>` released (see :ref:`Releasing the GIL <nogil>`).
+However, there is now some experimental support for running with the GIL in
+freethreading builds. You must either use these functions in a no-gil block,
```

Comment:
I guess that this wording will be confusing for most people that are not intimately familiar with how the free-threaded build works internally. Maybe we could reword it to something else like "with the thread state saved"? Although that seems suboptimal as well.

Ditto for other places.

Reply:
Yes good point.

I think we're stuck with the naming for the existing `with gil:`/`with nogil` features. But I can add an explanation if the free-threading docs about what that means the the context of a freethreading build.

But since this is a new feature I can probably pick a different name for the parameter. And I'll try to rephrase the docs to be a better clearer.